arbor/include/arbor/arbexcept.hpp

```diff
@@ -65,6 +66,13 @@ struct bad_event_time: arbor_exception {
     time_type sim_time;
 };
 
+// Morphology errors:
+
+struct no_such_segment: arbor_exception {
+    explicit no_such_segment(msize_t i);
+    msize_t index;
+};
+
```

Comment:
Shouldn't this go into `morphexcept.hpp`?

Reply:
Yes it should. I got myself confused by the two locations.